buildSrc/build.gradle.kts

```diff
@@ -69,4 +77,5 @@ dependencies {
     implementation("com.google.guava:guava:$guavaVersion")
     api("com.github.jk1:gradle-license-report:$licenseReportVersion")
     implementation("org.ajoberstar.grgit:grgit-core:${grGitVersion}")
+    implementation("net.ltgt.gradle:gradle-errorprone-plugin:${errorProneVersion}")
```

Comment:
One thing I would like to point out.
In order to configure `ErrorProne` in a type-safe manner we need it on a classpath of `buildSrc`.  Thus, it'd be on a classpath of each build file and no version should be specified in `plugins.apply` block.